### PR TITLE
feature/test ES256 verification

### PR DESCRIPTION
**Pull request by JGiter**
Added test of ES256 token

### CODE REVIEW

File: test/AuthTokenVerifier.test.ts [hunk 1]
```diff
@@ -119,16 +118,46 @@ describe("AuthTokenVerifier", () => {
...
+      //     id: `did:${Methods.Erc1056}:${delegate.address}#${PubKeyType.SignatureAuthentication2018}`,
+      //     type: PubKeyType.SignatureAuthentication2018,
+      //     publicKeyHex: delegate.publicKey,
+      //   } as IPublicKey);
+      //   verifier = new AuthTokenVerifier(document);
+      //   claim = await createClaim(delegate);
+      //   const did = await verifier.verify(claim);
+
+      //   assert.strictEqual(did, document.id);
+      // });
```
Comment (jrhender) [inline test/AuthTokenVerifier.test.ts:158]: Is this intended to be commented?
```suggestion
      // it("sigAuth delegate should be authenticated", async () => {
      //   const document = mockDocument(identity);
      //   document.publicKey.push({
      //     id: `did:${Methods.Erc1056}:${delegate.address}#${PubKeyType.SignatureAuthentication2018}`,
      //     type: PubKeyType.SignatureAuthentication2018,
      //     publicKeyHex: delegate.publicKey,
      //   } as IPublicKey);
      //   verifier = new AuthTokenVerifier(document);
      //   claim = await createClaim(delegate);
      //   const did = await verifier.verify(claim);

      //   assert.strictEqual(did, document.id);
      // });
```

**Review by jrhender — CHANGES_REQUESTED** [review]
Looks like a good start but not quite complete yet?

File: test/AuthTokenVerifier.test.ts (hunk 1 above)
Comment (JGiter) [reply test/AuthTokenVerifier.test.ts]: Sorry, it is left after debugging

Comment (jrhender) [reply test/AuthTokenVerifier.test.ts]: No problem. Please let me know when to review again

File: test/AuthTokenVerifier.test.ts [hunk 2]
```diff
@@ -117,18 +115,36 @@ describe("AuthTokenVerifier", () => {
...
       beforeAll(() => {
-        createClaim = createEthersClaim;
+        delegate = new Keys();
+        createClaim = createEIP191claim;
       });
       delegateTests();
     });
 
-    describe("With Json web signature", () => {
+    describe.only("With P256 signer", () => {
```
Comment (jrhender) [inline test/AuthTokenVerifier.test.ts:134]: Is this `.only` correct?
```suggestion
    describe("With P256 signer", () => {
```

**Review by jrhender — CHANGES_REQUESTED** [review]

**Review by jrhender — APPROVED** [review]

**Pull request merged**